tests/test_presence_do.py

```
@@ -0,0 +1,494 @@
+        })
+        return do, sid, ws
+
+    async def test_position_update_changes_presence(self):
+        do, sid, ws = await self._setup_user()
+        await do.on_webSocketMessage(ws, json.dumps({"type": "presence", "x": 0.8, "y": 0.2}))
+        assert do.presence["alice"]["x"] == pytest.approx(0.8)
+        assert do.presence["alice"]["y"] == pytest.approx(0.2)
+
+    async def test_only_x_changed(self):
+        """Sending only x must not modify y."""
+        do, sid, ws = await self._setup_user()
```

Ruff (0.15.10)

[warning] Unpacked variable `sid` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)